### PR TITLE
fix(gatsby-source-wordpress): Sync sample w/docs

### PR DESCRIPTION
**Pull request by DylanTackoor**
## Description

Updates example site gatsby-node.js to match documentation.

## Related Issues

Previously implemented in #11739.

### CODE REVIEW

File: examples/using-wordpress/gatsby-node.js [hunk 1]
```diff
@@ -1,108 +1,87 @@
...
+  // queries against the local WordPress graphql schema. Think of
+  // it like the site has a built-in database constructed
+  // from the fetched data that you can run queries against.
+  const result = await graphql(`
+    {
+      allWordpressPage {
+        edges {
+          node {
+            id
+            link
```
Comment (DSchau) [inline examples/using-wordpress/gatsby-node.js]: This should be slug right? _Or_ if it is link it should match the iteration in the forEach later when the node content is sourced--currently we're grabbing slug.

File: examples/using-wordpress/gatsby-node.js [hunk 2]
```diff
@@ -1,108 +1,87 @@
...
-      .then(result => {
-        if (result.errors) {
-          console.log(result.errors)
-          reject(result.errors)
+      }
+      allWordpressPost {
+        edges {
+          node {
+            id
+            link
```
Comment (DSchau) [inline examples/using-wordpress/gatsby-node.js]: Same here.

**Comment by DylanTackoor** [comment]
@DSchau anything else this needs?

File: examples/using-wordpress/gatsby-node.js [hunk 3]
```diff
@@ -1,108 +1,87 @@
...
-            edges {
-              node {
-                id
-                slug
-                status
-                template
-              }
-            }
+  // The “graphql” function allows us to run arbitrary
+  // queries against the local WordPress graphql schema. Think of
```
Comment (DSchau) [inline examples/using-wordpress/gatsby-node.js]:
```suggestion
  // queries against the local Gatsby GraphQL schema. Think of
```

File: examples/using-wordpress/gatsby-node.js [hunk 4]
```diff
@@ -1,108 +1,87 @@
...
-            })
-          })
-          resolve()
-        })
-      })
-    // ==== END POSTS ====
+  // Access query results via object destructuring
+  const { allWordpressPage, allWordpressPost } = result.data
+
+  // Create Page pages.
```
Comment (DSchau) [inline examples/using-wordpress/gatsby-node.js]:
```suggestion
```

Not really needed--the code is sorta self-documenting here, right?

**Comment by DylanTackoor** [comment]
Done :)

File: examples/using-wordpress/gatsby-node.js (hunk 4 above)
Comment (wardpeet) [reply examples/using-wordpress/gatsby-node.js]: @DylanTackoor this one is still here

Comment (DylanTackoor) [reply examples/using-wordpress/gatsby-node.js]: @wardpeet Whoops! Got it!

**Comment by sidharthachatterjee** [comment]
Published in gatsby-source-wordpress@3.0.42

**Pull request merged**